### ISSUE TITLE
Where are the D communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/d#66]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is D supported by one or more large organizations?
* Does D have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about D? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for D? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in D? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in D?
* Are there popular newsletters for D?
* Is D taught at programming bootcamps? (If so, what are some examples?)
* Is D taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about D?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by mdparker** [comment]
> Is D supported by one or more large organizations?

Large ones, no. The primary supporter of D is the [D Language Foundation](https://dlang.org/foundation/about.html). A number of companies using D support the language in various forms (monetary donations, open source, conference organization, etc..). Including, but not limited to, [Sociomantic Labs](https://www.sociomantic.com/), [Weka.IO](https://www.weka.io/), [Symmetry Investments](http://symmetryinvestments.com/). [Funkwerk](http://www.funkwerk.com/en/).

> Does D have an official community manager?

Officially, no. But there are a handful of people who wear that hat (myself included).

> Do you know of specific communities (online or offline) that are enthusiastic about D? (Chat communities, forums, meetups, student clubs, etc)

[The D Forums](https://forum.dlang.org/)
The [#d IRC channel at freenode.net](irc://irc.freenode.net/d)
The [d_language subreddit](https://www.reddit.com/r/d_language/)
Various [meetups around the globe](https://www.meetup.com/topics/dpl/)

> Are there popular conferences for D? (If so, what are some examples?)

We recently wrapped up [the 6th annual DConf](http://dconf.org/2018/index.html)

>Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in D? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)

See the next answer.

> Are there specific groups or programs dedicated to mentoring people in D?

The D Language Foundation [provides scholarships](https://dlang.org/foundation/upb-scholarship.html) for graduate-level (MSc, PhD) research at the University "Politehnica" Bucharest (UPB). The students receive mentoring from the Foundation. Beyond that, I'm not aware of anything public, though I am aware of at least one unannounced program currently in development.

> Are there popular newsletters for D?

Not that I'm aware of.

> Is D taught at programming bootcamps? (If so, what are some examples?)

I haven't heard of any.

> Is D taught at universities? (If so, what are some examples?)

I know it *has* been taught [at the University of Minnesota](https://forum.dlang.org/post/yutwxozxoqrfalzfswht@forum.dlang.org) and [Utah Valley University](http://dconf.org/2015/talks/gubler.html). 

Hope this helps!

**Comment by p0nce** [comment]
There is also the D Discord which is afaik as active than the IRC server https://discord.gg/bMZk9Q4

**Comment by kytrinyx** [comment]
This is great information, thanks!

**Comment by kytrinyx** [comment]
I've got what I need for now; closing this out. (Should you come across any other good sources, feel free to add them here, I'll be coming back to this). 🌼  Thanks!
